tutorials/cookbooks/running_production.md

```diff
@@ -0,0 +1,308 @@
+
+> Reverse proxies are very useful for security, performance, scalability and logging reasons. If you are new to Reverse proxies, you may enjoy this article explaining the why and the how of [reverse proxies](https://www.keycdn.com/support/nginx-reverse-proxy)  
+
+Making Nginx work as a proxy server is really simple. First of all we need to install it in our machine.
+
+```bash
+# Install Nginx on Debian
+$ apt-get install nginx -y
+```
+
+Now we need to delete its default configuration file in order to add our own. This is important because the default port for HTTP, the `port 80` is used by nginx by default and if we try to use it for MeiliSearch we will create a conflict. We will replace its default file by our own. You can also make MeiliSearch listen to another port by specifying it in the nginx configuration file, but we will not cover this option in this article.
+
+```bash
+# Delete the default configuration file for Nginx
+$ rm -f /etc/nginx/sites-enabled/default
+
+# Add our configuration file, specifyind the Reverse Proxy settings
+$ cat << EOF > /etc/nginx/sites-enabled/meilisearch
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
+    server_name _;
+    location / {
+        proxy_pass  http://127.0.0.1:7700;
+    }
+}
+EOF
+```
+
+finally we need to enable and start the nginx service as we did before, to make sure it is always available
+
+```bash
+# Reload the operating system daemons / services
+$ systemctl daemon-reload
+
+# Enable and start nginx service
+$ systemctl enable nginx
+$ systemctl restart nginx
+```
+
+Our MeiliSearch is now up, deployed in a production environment, using a safe API key, and being served by a Reverse Proxy Nginx. We should now be able to do requests to our server from the outside world by doing any HTTP request MeiliSearch accepts at our own IP address (http://<your-ip-address>). The IP address is the same you used to connect to your machine via SSH in Step 1.
+
+
+> If you want to learn more about using Nginx as a Reverse Proxy, see its dedicated documentation [Nginx as a Reverse Proxy](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/)
+
+Our only problem now is that we are using MeiliSearch under HTTP, without any additional security. This is an important issue, as the content that is being passed via HTTP can easily be read or modified by attackers, and someone could get full or partial acces to your data. In order to prevent this, it's important to use the SSL protocol, which will let us use HTTPS, a verification and encrypted way of transmitting data.
```

Comment:
I changed "verification" but I am not sure. Did you mean "a SSL/TLS certificate" or am I wrong?
```suggestion
The only remaining problem is that MeiliSearch processes requests via HTTP without any additional security. The content that is being transmitted over HTTP could easily be read or modified by attackers, and someone could get full or partial access to your data. In order to prevent this to happen, it's important to use the SSL protocol, which will enable you to use HTTPS, a SSL/TLS certificate, and securely transmit data.
```

Reply:
```suggestion
The only remaining problem is that MeiliSearch processes requests via HTTP without any additional security. The content that is being transmitted over HTTP could easily be read or modified by attackers, and someone could get full or partial access to your data. In order to prevent this to happen, it's important to use the HTTPS, which will enable you to use a SSL/TLS certificate, and securely transmit data.
```

Reply:
Good for me